### PR TITLE
Experimental Deepgram streaming integration

### PR DESCRIPTION
**Pull request by twschiller**
## What does this PR do?

- Experimental Deepgram streaming integration
- Adds a `capture` platform protocol
  - Refactors screenshot tab to use the platform protocol (decouples the brick from the content script)
- Add bricks
  - [Experimental] Capture Audio
  - [Experimental] Stop Audio Capture
- Refactors offscreen document code to split out creation code vs. the actual document

## Remaining Work

- [x] Verify manifest change doesn't require extension re-activation

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: scripts/__snapshots__/manifest.test.js.snap [hunk 1]
```diff
@@ -125,7 +126,7 @@ exports[`customizeManifest release builds beta 1`] = `
     "managed_schema": "managedStorageSchema.json",
   },
   "version": "1.8.13.3000",
-  "version_name": "1.8.13",
+  "version_name": "1.8.13.3000-alpha+undefined",
```
Comment (grahamlangford) [inline scripts/__snapshots__/manifest.test.js.snap:129]: Not sure why this line changed, might fail in pipeline again?

Comment (twschiller) [reply scripts/__snapshots__/manifest.test.js.snap:129]: Good catch -- my local environment has some differences

File: src/background/messenger/api.ts [hunk 2]
```diff
@@ -40,7 +40,7 @@ export const traces = {
   clearAll: getNotifier("CLEAR_ALL_TRACES", bg),
 };
 
-export const captureTab = getMethod("CAPTURE_TAB", bg);
+export const captureTabScreenshot = getMethod("CAPTURE_TAB_SCREENSHOT", bg);
```
Comment (twschiller) [inline src/background/messenger/api.ts:43]: Consider moving this method into tabCapture in the future

File: src/tinyPages/offscreen.ts [hunk 3]
```diff
@@ -161,12 +71,159 @@ export const sendErrorViaErrorReporter = async (
...
+  }
+
+  mediaClient = null;
+  markRecordingTab(null);
 }
+
+// Use optional chaining in case the chrome runtime is not available:
+// https://github.com/pixiebrix/pixiebrix-extension/issues/8397
+chrome.runtime?.onMessage?.addListener(async (message: unknown) => {
+  if (isRecordErrorMessage(message)) {
```
Comment (grahamlangford) [inline src/tinyPages/offscreen.ts:220]: NIT: A `switch(true) {...}` might be more readable here

**Review by grahamlangford — APPROVED** [review]
Small style NIT. Since this is mostly a POC, I don't think it's very important

**Pull request merged**